tests/gold_tests/headers/gold/bad_good_request_v10.gold

```diff
@@ -0,0 +1,22 @@
+HTTP/1.0 400 Invalid HTTP Request
+Date: ``
+Server: ATS/``
+Cache-Control: no-store
+Content-Type: text/html
+Content-Language: en
+Content-Length: 219
+
+<HTML>
+<HEAD>
+<TITLE>Bad Request</TITLE>
+</HEAD>
+
+<BODY BGCOLOR="white" FGCOLOR="black">
+<H1>Bad Request</H1>
+<HR>
+
+<FONT FACE="Helvetica,Arial"><B>
+Description: Could not process this request.
+</B></FONT>
+<HR>
+</BODY>
```

Comment:
I wonder whether this gold file is verifying too much? Any small changes in this output, even by one character, will cause it to fail. Perhaps:

```
HTTP/1.0 400 Invalid HTTP Request
Date: ``
Server: ATS/``
Cache-Control: no-store
Content-Type: text/html
Content-Language: en
Content-Length: ``

``
<TITLE>Bad Request</TITLE>
``
Description: Could not process this request.
``
```

Reply:
I agree to a certain extent, but the test is also ensuring that the following request is not processed which the wildcard at the end would not catch.  Guess I could also do this via Contains and Excludes tests.

Reply:
But the problem of the excludes test is if the following request is processed in a different way than I would expect, the excludes test would not catch it.

Reply:
Ah, OK. That makes sense.